calico-enterprise_versioned_docs/version-3.17/releases.json

```diff
@@ -1,4 +1,271 @@
+        "version": "v3.17.2"
+      },
+      "ingress-collector": {
+        "image": "tigera/ingress-collector",
+        "version": "v3.17.2"
+      },
+      "l7-collector": {
+        "image": "tigera/l7-collector",
+        "version": "v3.17.2"
+      },
+      "license-agent": {
+        "image": "tigera/license-agent",
+        "version": "v3.17.2"
+      },
+      "linseed": {
+        "image": "tigera/linseed",
+        "version": "v3.17.2"
+      },
+      "tigera-cni": {
+        "image": "tigera/cni",
+        "version": "v3.17.2"
+      },
+      "firewall-integration": {
+        "image": "tigera/firewall-integration",
+        "version": "v3.17.2"
+      },
+      "egress-gateway": {
+        "image": "tigera/egress-gateway",
+        "version": "v3.17.2"
+      },
+      "honeypod": {
+        "image": "tigera/honeypod",
+        "version": "v3.17.2"
+      },
+      "honeypod-exp-service": {
+        "image": "tigera/honeypod-exp-service",
+        "version": "v3.17.2"
+      },
+      "honeypod-controller": {
+        "image": "tigera/honeypod-controller",
+        "version": "v3.17.2"
+      },
+      "key-cert-provisioner": {
+        "image": "tigera/key-cert-provisioner",
+        "version": "v1.1.10",
+        "registry": "quay.io"
+      },
+      "anomaly_detection_jobs": {
+        "image": "tigera/anomaly_detection_jobs",
+        "version": "v3.17.2"
+      },
+      "anomaly-detection-api": {
+        "image": "tigera/anomaly-detection-api",
+        "version": "v3.17.2"
+      },
+      "elasticsearch-metrics": {
+        "image": "tigera/elasticsearch-metrics",
+        "version": "v3.17.2"
+      },
+      "packetcapture": {
+        "image": "tigera/packetcapture",
+        "version": "v3.17.2"
+      },
+      "prometheus": {
+        "image": "tigera/prometheus",
+        "version": "v3.17.2"
+      },
+      "coreos-prometheus": {
+        "version": "v2.47.0"
+      },
+      "coreos-prometheus-operator": {
+        "version": "v0.62.0"
+      },
+      "coreos-config-reloader": {
+        "version": "v0.62.0"
+      },
+      "prometheus-operator": {
+        "image": "tigera/prometheus-operator",
+        "version": "v3.17.2"
+      },
+      "prometheus-config-reloader": {
+        "image": "tigera/prometheus-config-reloader",
+        "version": "v3.17.2"
+      },
+      "tigera-prometheus-service": {
+        "image": "tigera/prometheus-service",
+        "version": "v3.17.2"
+      },
+      "es-gateway": {
+        "image": "tigera/es-gateway",
+        "version": "v3.17.2"
+      },
+      "deep-packet-inspection": {
+        "image": "tigera/deep-packet-inspection",
+        "version": "v3.17.2"
+      },
+      "eck-elasticsearch-operator": {
+        "version": "2.6.1"
+      },
+      "elasticsearch-operator": {
+        "image": "tigera/eck-operator",
+        "version": "v3.17.2"
+      },
+      "coreos-alertmanager": {
+        "version": "v0.25.1"
```

Comment:
I noticed today that this should actually be v0.25.0

Reply:
Fixed.